recipes/cpptango/build.sh

```diff
@@ -0,0 +1,16 @@
+mkdir build
+cd build
+cmake -DCMAKE_BUILD_TYPE=Relase \
```

Comment:
Thanks for the typo!
I thought it makes more sense to build in release mode when creating a package. This is what is done for most Conda packages as far as I know.
After I guess it depends what the usage will be. What could maybe be possible is to create 2 outputs: `cpptango` and `cpptango-debug`. To build in debug mode, do we need to have the dependent libraries available in debug as well or not? On windows I noticed that in debug mode, it's looking for debug libraries. They won't be available with Conda.

Reply:
For PyTango debugging, testing and development (which is done via Conda on Linux), it is very useful to have a debug version of cppTango.  I'm not sure if any sites are planning to do production deployments via Conda.  Production deployments may benefit from the increased performance from a release mode build.

If we can only have one package, then it should be debug (at least for Linux).  If we can have multiple (and it is easy to maintain), then adding `cpptango-debug` would be OK.

Reply:
After looking a bit more, it might not be that simple. Just creating 2 cppTango packages would make it a bit difficult for the other packages (we would need one pytango-debug depending on cpptango-debug...). From what I read, it could be achieved with track features.
Anyway, it's probably not worth the effort for now.
I switched to `Debug` type. We can always investigate later if we do want a `Release` one.